src/components/home/ContactsList.tsx

```diff
@@ -27,18 +30,26 @@ const ContactsList = () => {
       <div>{contact.state}</div>
       <div>{contact.zip}</div>
     </div>
-  ))
+  ));
 
   useEffect(() => {
-    dispatch(getAllContacts())
+    dispatch(getAllContacts());
   }, []);
 
+  useEffect(() => {
```

Comment:
I don't think this second useEffect is needed, besides troubleshooting. Am I good to remove this?

Reply:
Yeah definitely good to remove it